pkg/proxy/iptables/proxier.go

```diff
@@ -1110,7 +1110,7 @@ func (proxier *Proxier) syncProxyRules() {
 				} else {
 					socket, err := proxier.portMapper.OpenLocalPort(&lp)
 					if err != nil {
-						msg := fmt.Sprintf("can't open %s, skipping this externalIP: %v", lp.String(), err)
+						msg := fmt.Sprintf("can't open port %s, skipping it: %v", lp.String(), err)
```

Comment:
I am so sorry to pick on this so hard.  Since you posted the output (THANK YOU) I see the error is reported twice.  I think we do not need the `%v` and `err` in msg.  That makes it about the same as the log-line

Reply:
OK, OK, fixed.
```
$ kubectl get event
LAST SEEN   TYPE      REASON                                            OBJECT           MESSAGE
4s          Warning   listen tcp4 :30486: socket: too many open files   node/127.0.0.1   can't open port "nodePort for default/temp-svc:503" (:30486/tcp4), skipping it
```